File: snippets/eintrittspreise.py
age = 18

if age <= 6 or age >= 60:
    print(f"Der Eintritt ist für dich kostenlos")
elif age >= 7  and age <= 17:
    print (f"Der Eintritt kostet für dich: 2,50€")
elif age > 17 and age <= 30:
    print (f"Der Eintritt kostet für dich: 3€")
else:
    print(f"Der Eintritt kostet für dich: 3,50€")

File: beispiele/eintrittspreise/eintrittspreise.php
<?php
$age = 18;

if ($age <= 6 || $age >= 60) {
    echo "Eintritt kostenlos!";
} elseif ($age >= 7 && $age <= 18) {
    echo "Eintritt: 2,50€";
} elseif ($age > 18 && $age <= 30) {
    echo "Eintritt: 3€";
} else {
    echo "Eintritt: 3,50€";
}
?>